.dockerignore

```diff
@@ -1,5 +1,3 @@
-.git/
```

Comment:
Don't have time to do a full review right now but, as a quick drive by comment, this doesn't look quite right to me. Doesn't it mean we end up bundling the entire git repo into the Docker image? Which I'm pretty sure we don't want to do.

Reply:
It's bind mounted at build time on line 33 of the diff

Reply:
Yes, it's bind-mounted: we need to make the repo available, to extract the version from a tag (see the [_Usage from Docker_][1] section of `setuptools_scm`'s documentation).

[1]: https://github.com/pypa/setuptools_scm/#usage-from-docker

Reply:
Ah yes. And we never do `COPY .` so it wouldn't get included anyway. Ignore me!